### PR TITLE
More tips of the day

### PR DESCRIPTION
**Pull request by nexusmrsep**
#### Summary

```SUMMARY: Content "New snippets for tips of the day."```

#### Purpose of change
Giving more hints about existing convoluted mechanics and overall tips that are not too spoilery.

#### Describe the solution
More tips. Some are mine, and some were gathered here: https://discourse.cataclysmdda.org/t/tip-of-the-day-suggestions/16665 They were despoilerized and generalized. We don't want to give out too much or keep players by their hands, just a poke in the right direction. 

#### Describe alternatives you've considered
Keep'em coming. 

#### Additional context
I want to thank our community for giving great ideas for the tips on the Discourse thread.
That is still a valid place for new ideas.

### CODE REVIEW

File: data/core/tips.json [hunk 1]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't overlook protective gear, like gas masks or turnout gear.  You never know." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Out of resources?  Breaking or disassembling stuff can yield intresting components." ]
```
Comment (Inglonias) [inline data/core/tips.json]: Typo. The word should be "interesting" instead of "intresting"

**Comment by Marrim** [comment]
>    "text": [ "Famous last words:  Lets poke this electric zombie with a metal stick." ]

Occurs twice, and the first time there is a typo.

File: data/core/tips.json [hunk 2]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "A full stomach will not solve your starvation.  Give it time and eat regulary." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Be mindful of enviromental protection of your gear.  It can take you places." ]
```
Comment (Marrim) [inline data/core/tips.json]: Typo: "environmental".

File: data/core/tips.json [hunk 3]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Like to gamble?  Use mutagen!  Be a winner today!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Famous last words:  Lets poke this electiric zombie with a metal stick." ]
```
Comment (Marrim) [inline data/core/tips.json]: Typo: "electric".
Also this line is repeated again below.

File: data/core/tips.json [hunk 4]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Food's shelf life is not given.  Storage conditions matter, especially temperature." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Frozen food doesn't rot, but may be inedible if not deforsted." ]
```
Comment (Marrim) [inline data/core/tips.json]: Typo: "defrosted" is the correct spelling.

File: data/core/tips.json [hunk 5]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Some dry  food can be eaten frozen, while other may become mushy after defrosting." ]
```
Comment (Marrim) [inline data/core/tips.json]: Double space before "food".
Also better to write "some foods" and "while others".

File: data/core/tips.json [hunk 6]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There are fungi you want to eat, and those you want to burn.  You'll know the difference." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Having a backup weapon is thoughtful.  As well as having a backup plan." ]
```
Comment (Leland) [inline data/core/tips.json]: Would recommend

> Having a backup weapon is wise.  So is having a backup plan.

File: data/core/tips.json (hunk 3 above)
Comment (Leland) [reply data/core/tips.json]: Also, should be **let's**

File: data/core/tips.json [hunk 7]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Why it's called chicken walker?  Because you'll chicken out every time you'll see it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Proceed with alertness and extreme prejudice.  Routine kills." ]
```
Comment (Leland) [inline data/core/tips.json]: I don't know what this tip means, frankly

File: data/core/tips.json [hunk 8]
```diff
@@ -173,5 +173,120 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Famous last words:  Lets poke this electric zombie with a metal stick." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "A full stomach will not solve your starvation.  Give it time and eat regulary." ]
```
Comment (Leland) [inline data/core/tips.json]: regularly

File: data/core/tips.json (hunk 5 above)
Comment (Leland) [reply data/core/tips.json]: Also, "can be eaten frozen" and "may become mushy after defrosting" are not contrasting things. What's probably meant to be said here is:

`Though some kinds of dry food can be frozen and defrosted without an issue...` etc

**Comment by nexusmrsep** [comment]
Thanks for pointing those things out, they are fixed now. If you see any more tell me and I'll get them fixed.

**Pull request merged**